apps/src/code-studio/pd/workshop_dashboard/components/survey_results_header.jsx

```diff
@@ -140,6 +141,14 @@ const SurveyResultsHeader = React.createClass({
     this.setSurveyPanel(this.state.selectedCourse, event.target.value, event.target.selectedOptions[0].innerHTML);
   },
 
+  handleOnClickDownloadCsv() {
+    if (this.props.organizerView && !this.state.selectedWorkshopId) {
+      window.open(`/api/v1/pd/workshop_organizer_survey_report_for_course/${this.state.selectedCourse}.csv`);
+    } else if (this.state.selectedWorkshopId) {
+      window.open(`/api/v1/pd/workshops/${this.state.selectedWorkshopId}/workshop_survey_report.csv`);
+    }
```

Comment:
These base URLs (e.g. `/api/v1/pd/workshops/`) are now used each in 2 places (here and in the AJAX call below). They should be constants defined in one place.